revolv/templates/base/home.html

```diff
@@ -60,7 +60,14 @@
                 <div class="learn-more button">Learn how RE-volv works</div>
             </a>
         </div>
-        <video autoplay loop muted src="{% static "videos/cover.mp4" %}" class="cover-video"></video>
+        <!-- Background video to play on desktop view -->
+        <video autoplay loop muted poster="{% static "images/cover-video-poster.png" %}" class="cover-video">
+            <source src="{% static "videos/cover.mp4" %}" type="video/mp4">
+            <source src="{% static "videos/cover.webm" %}" type="video/webm">
+        </video>
+        <!-- Background image displayed on mobile devices -->
+        <img src="{% static "images/mobile-cover.png" %}" 
```

Comment:
This is a super nitpick, but we don't really need this img tag to be on separate lines when it's shorter than some elements which are on one line above